plugins/justify/plugin.js

```diff
@@ -150,6 +149,12 @@
 					if ( block.isReadOnly() )
 						continue;
 
+					// #16808 Check if style or class might be applied to currently processed element
+					var tag = block.getName(),
+						isAllowedTextAlign, isAllowedCssClass;
+					isAllowedTextAlign = editor.filter.check( tag + '{text-align}' ) || false;
+					isAllowedCssClass = editor.filter.check( tag + '(' + cssClassName + ')' ) || false;
```

Comment:
Can't we early return from function if both `isAllowedTextAlign` and `isAllowedCssClass` are false?

Also I'd check if default value is needed as `editor.filter.check` is supposed to return boolean.

Mind also the fact that below you use `editor.activeFilter` instead of `editor.filter`. Probably it would be a better idea to reuse it here.